CIP-xxxx/README.md

```diff
@@ -0,0 +1,127 @@
+---
+CIP: ?
+Title: SRV registry
+Status: Proposed
+Category: Network
+Authors:
+  - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+Implementors: N/A
+Discussions: 
+  - [PR](https://github.com/cardano-foundation/CIPs/pull/1033)
+Created: 2025-04-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines the procedure for assigning Service Record (SRV) prefixes for Cardano applications (like `cardano-node`, `mithril`, etc.).
+
+This CIP constructs means of sharing authoritative information between SPOs who deploy services and DAP developers who write them.
+
+## Motivation: why is this CIP necessary?
+
+The **Cardano Ledger** allows the use of SRV records in the **SPO** registration certificate (see [register-stake-pool]).
+
+In this CIP, we propose to make them usable by decentralised applications running on `Cardano`.
+
+Having access to the ledger (either directly or through tools like `cardano-cli`) will allow decentralised applications (DAPS) developers to construct networks based on registered relays. 
+
+All involved parties (SPOs, DAP developers, etc) need to agree on SRV prefixes used by various DAPs and thus a public registry governed by a public CIP process is required to foster DAPs that co-exist with the **Cardano** network, like **Mithril** (see [CIP#137]).
+
+Note that the notion of a DAP is specific in this context: these are applications, which construct their own network and relay on ledger peers published on the block-chain.   This include node implemtations, or 2nd-layer solutions like Mithril or Hydra.
```

Comment:
```suggestion
Note that the notion of a DAP is specific in this context: these are applications, which construct their own network and relay on ledger peers published on the block-chain. This include node implementations, or 2nd-layer solutions like Mithril or Hydra.
```

fixed spelling and removed extra space

Reply:
It's a writing style to use double spaces between sentences.  It might be in other places...

Reply:
@coot it's in other places & we have agreed (after last confirmed with @perturbing) that both double & single spaces after a full stop are used routinely in CIPs and acceptable either (or both) ways.  I think @Ryun1 had tagged the _triple_ space and then suggested removing one too many of the extra spaces 😅